xcode/Subconscious/Shared/Components/Common/Profile/EditProfileSheet.swift

```diff
@@ -89,7 +89,7 @@ struct EditProfileSheetModel: ModelProtocol {
     var bioField: FormField<String, String> = FormField(
         value: "",
         validate: { value in
-            value.count >= 280 ? nil : value
+            value.count > 280 ? nil : value
```

Comment:
Off by one 🥲

Reply:
Should this be factored out into a function, and add a regression test?

`Prose.swift` is a good spot for stuff like this. We may want to add a constant or int-valued enum (something like `bioCount` or `shortTextCount`), and a utility fn (e.g. `isShortText(_ text: String)`).

Non-blocking.